app.js

```diff
@@ -1,25 +1,29 @@
 const express = require('express');
 
-// const connect = require('./database.js');
+const db = require('./database.js');
 
 const app = express();
 
 app.get('/',function(request, response){
 	response.send('Hello World');
 })
 
-app.get('/student',function(req, res){
+app.get('/blogs',function(req, res){
 	res.send({name: ['Ram',"Shyam"],roll:3});
+
 })
 
-app.post('/students',function(request, response){
+app.post('/blogs',function(request, response){
 	student = request.body;
```

Comment:
1. use body-parser npm package to read the body data from HTTP request. ref: https://www.npmjs.com/package/body-parser
2. save the info to db ref: https://mongoosejs.com/docs/models.html#constructing-documents